core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -262,7 +262,7 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging with Seria
   /**
    * Get a time parameter as seconds; throws a NoSuchElementException if it's not set. If no
    * suffix is provided then seconds are assumed.
-   * @throws java.util.NoSuchElementException
+   * @throws java.util.NoSuchElementException If the time parameter is not set
```

Comment:
And.. here is my observation so far.

| Link style                                  | Scaladoc       | Javadoc     | Link          | Error/Warn | etc.                                                |
| ------------------------------------------- | -------------- | ----------- | ------------- | ---------- | --------------------------------------------------- |
| `DataFrame`                                 | Renders OK     | Renders OK  | X             | X          |                                                     |
| `[[DataFrame]]`                             | Renders OK     | Renders OK  | Scaladoc only | Error      | Fixed in https://github.com/apache/spark/pull/16013 |
| `@throws IllegalArgumentException` ...         | Renders OK     | Renders OK  | X             | Warn       |                                                     |
| `@throws java.util.NoSuchElementException` ...  | Renders OK     | Renders OK  | X             | Warn       |                                                     |
| `@throws java.util.NoSuchElementException` [NO DESCRIPTION]  | X     | Renders OK  | X             | Warn       | Fixed here                                              |